src/SFA.DAS.AssessorService.Database/StoredProcedures/Load_OFQUAL_Standards.sql

```diff
@@ -50,7 +50,26 @@ BEGIN
             MERGE INTO [dbo].[OfqualOrganisation] tar
             USING 
             (
-                SELECT soo.*, @dateTimeUtc [LastUpdated]
+                -- providing a default min date for OfqualRecognisedFrom when no value is present in the staging table
+                -- this is temporary workaround for non-nullable columns which will be investigated separately
+                SELECT
+                   soo.[RecognitionNumber]
+                  ,[Name]
+                  ,[LegalName]
+                  ,[Acronym]
+                  ,[Email]
+                  ,[Website]
+                  ,[HeadOfficeAddressLine1]
+                  ,[HeadOfficeAddressLine2]
+                  ,[HeadOfficeAddressTown]
+                  ,[HeadOfficeAddressCounty]
+                  ,[HeadOfficeAddressPostcode]
+                  ,[HeadOfficeAddressCountry]
+                  ,[HeadOfficeAddressTelephone]
+                  ,[OfqualStatus]
+                  ,ISNULL([OfqualRecognisedFrom], '1900-01-01 00:00:00.000') OfqualRecognisedFrom
```

Comment:
Will it be an issue that the time has 7 digits after the dot instead of 3? As in `2013-09-17 00:00:00.0000000`

Reply:
I don't think so - it seems to work anyway